analyzers/its/expected/Nancy/Nancy-{34576216-0DCA-4B0F-A0DC-9075E75A676F}-S4790.json

```diff
@@ -4,45 +4,6 @@
 "id":  "S4790",
 "message":  "Make sure this weak hash algorithm is not used in a sensitive context here.",
 "location":  {
-"uri":  "sources\Nancy\src\Nancy\Cryptography\DefaultHmacProvider.cs",
```

Comment:
These are the lines which are now compliant:
```
private readonly int hmacLength = new HMACSHA256().HashSize / 8;

var hmacGenerator = new HMACSHA256(this.key);

var algorithm = new SHA256Managed();
```

they are all related to SHA 256 which is considered secure.